### PR TITLE
feat: drop python2 support

### PR DESCRIPTION
**Pull request by yihong0618**
fix: #231 
@jsamoocha @lwasser @hozn

### CODE REVIEW

**Review by jsamoocha — APPROVED** [review]
LGTM, thanks!

File: requirements.txt [hunk 1]
```diff
@@ -2,7 +2,6 @@ arrow
 requests>=2.0,<3.0dev
 units
 pytz
-six
```
Comment (lwasser) [inline requirements.txt:5]: oh good i just asked about dropping six i'll do that and see if it fixes the dep resolve on test pypi in a conda envt

**Review by lwasser — APPROVED** [review]

**Comment by lwasser** [comment]
ya'll this looks good to me! i'm not a python 2.x expert so if tests are running and you both think it's good let's merge. i'm hoping dropping six will help with the envt issue i'm having now in my test pypi push.

**Comment by hozn** [comment]
LGTM, thank you!  Maybe long overdue.  (With python 3.7+ things like typehints become nice options available too.)

**Comment by lwasser** [comment]
Ok merging!!

**Comment by lwasser** [comment]
Thank you everyone for the effort on this. i'll now bring it in to my package build workflow.

**Pull request merged**